package.json

```diff
@@ -154,6 +154,7 @@
     "lodash": "3.10.1",
     "lru-cache": "4.1.1",
     "markdown-it": "^8.4.1",
+    "markdown-to-jsx": "^6.6.8",
```

Comment:
I tried to use our own Markdown component, but it won't work for our use cases because it renders into a new element, whereas `markdown-to-jsx` seems to render into a fragment.

Reply:
By "our own Markdown component" I assume you mean `react-markdown` which is listed below? If so, have you tried configuring its `root` element type as in

```javascript
const markdownRenderers = {
  root: React.Fragment,
}
const text = <ReactMarkdown renderers={markdownRenderers}>{input}</ReactMarkdown>
```

Reply:
Ah, I meant the Markdown component at `ui/markdown/markdown`. I haven't seen `react-markdown`, but I'll try out your suggestion. Thanks!